src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Interop/msquic_generated.cs

```diff
@@ -0,0 +1,2677 @@
+                return ref MemoryMarshal.GetReference(MemoryMarshal.CreateSpan(ref Anonymous.STOP_COMPLETE, 1));
+            }
+        }
+
+        [StructLayout(LayoutKind.Explicit)]
+        internal partial struct _Anonymous_e__Union
+        {
+            [FieldOffset(0)]
+            [NativeTypeName("struct (anonymous struct)")]
+            public _NEW_CONNECTION_e__Struct NEW_CONNECTION;
+
+            [FieldOffset(0)]
+            [NativeTypeName("struct (anonymous struct)")]
+            public _STOP_COMPLETE_e__Struct STOP_COMPLETE;
+
+            internal unsafe partial struct _NEW_CONNECTION_e__Struct
+            {
+                [NativeTypeName("const QUIC_NEW_CONNECTION_INFO *")]
+                public QUIC_NEW_CONNECTION_INFO* Info;
+
+                [NativeTypeName("HQUIC")]
+                public QUIC_HANDLE* Connection;
+            }
+
+            internal partial struct _STOP_COMPLETE_e__Struct
+            {
+                public byte _bitfield;
+
+                [NativeTypeName("BOOLEAN : 1")]
+                public byte AppCloseInProgress
+                {
+                    get
+                    {
+                        return (byte)(_bitfield & 0x1u);
+                    }
+
+                    set
+                    {
+                        _bitfield = (byte)((_bitfield & ~0x1u) | (value & 0x1u));
+                    }
+                }
+
+                [NativeTypeName("BOOLEAN : 7")]
+                public byte RESERVED
+                {
+                    get
+                    {
+                        return (byte)((_bitfield >> 1) & 0x7Fu);
+                    }
+
+                    set
+                    {
+                        _bitfield = (byte)((_bitfield & ~(0x7Fu << 1)) | ((value & 0x7Fu) << 1));
+                    }
+                }
+            }
+        }
+    }
+
+    internal enum QUIC_CONNECTION_EVENT_TYPE
+    {
+        CONNECTED = 0,
+        SHUTDOWN_INITIATED_BY_TRANSPORT = 1,
+        SHUTDOWN_INITIATED_BY_PEER = 2,
+        SHUTDOWN_COMPLETE = 3,
+        LOCAL_ADDRESS_CHANGED = 4,
+        PEER_ADDRESS_CHANGED = 5,
+        PEER_STREAM_STARTED = 6,
+        STREAMS_AVAILABLE = 7,
+        PEER_NEEDS_STREAMS = 8,
+        IDEAL_PROCESSOR_CHANGED = 9,
+        DATAGRAM_STATE_CHANGED = 10,
+        DATAGRAM_RECEIVED = 11,
+        DATAGRAM_SEND_STATE_CHANGED = 12,
+        RESUMED = 13,
+        RESUMPTION_TICKET_RECEIVED = 14,
+        PEER_CERTIFICATE_RECEIVED = 15,
+    }
+
+    internal partial struct QUIC_CONNECTION_EVENT
+    {
+        public QUIC_CONNECTION_EVENT_TYPE Type;
+
+        [NativeTypeName("QUIC_CONNECTION_EVENT::(anonymous union)")]
+        public _Anonymous_e__Union Anonymous;
+
+        public ref _Anonymous_e__Union._CONNECTED_e__Struct CONNECTED
+        {
+            get
+            {
+                return ref MemoryMarshal.GetReference(MemoryMarshal.CreateSpan(ref Anonymous.CONNECTED, 1));
+            }
+        }
```

Comment:
Unrelated: I wonder why this can't be just 
```cs
        public ref _Anonymous_e__Union._CONNECTED_e__Struct CONNECTED => ref Anonymous.CONNECTED;
```

Reply:
This is auto generated by ClangSharp. I think there are some lifetime reasons that doesn't work.